docs/user/README.md

```diff
@@ -1,44 +1,67 @@
+To contact the Glean team you can:
 
-To contact us you can:
 - Find us in the [#glean channel on chat.mozilla.org](https://chat.mozilla.org/#/room/#glean:mozilla.org).
-- To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean%3A+SDK&priority=P3&status_whiteboard=%5Btelemetry%3Aglean-rs%3Am%3F%5D).
 - Send an email to *glean-team@mozilla.com*.
 - The Glean SDK team is: *:janerik*, *:dexter*, *:travis*, *:mdroettboom*, *:gfritzsche*, *:chutten*, *:brizental*.
 
-The source code is available [on GitHub](https://github.com/mozilla/glean/).
+![Glean logo](glean.jpeg)
+
+There are two implementations of Glean, with support for 5 different programming languages in total. Both implementations strive to contain the same features with similar, but idiomatic APIs.
+
+Unless clearly stated otherwise, regard the text in this book as valid for both clients and all the supported programming languages and environments.
+
+### [The Glean SDK](https://github.com/mozilla/glean)
+
+The Glean SDK is an implementation of Glean in Rust, with language bindings for **Kotlin**, **Python**, **Rust** and **Swift**.
+
+For development documentation on the `Glean SDK`, refer to [the Glean SDK development book](https://mozilla.github.io/glean/).
+
+To report issues or request changes on the Glean SDK, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean%3A+SDK&priority=P3&status_whiteboard=%5Btelemetry%3Aglean-rs%3Am%3F%5D).
+
+### [Glean.js](https://github.com/mozilla/glean.js)
+
+Glean.js is an implementation of Glean in **Javascript**. Currently, it only has support for usage in web extensions.
+
+For development documentation on `Glean.js`, refer to [the Glean.js development documentation](https://github.com/mozilla/glean.js/tree/main/docs).
+
+To report issues or request changes on the Glean SDK, file a bug in [Bugzilla in Data Platform & Tools :: Glean.js](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean.js&priority=P4&status_whiteboard=%5Btelemetry%3Aglean-js%3Am%3F%5D).
+
+> **Note** Glean.js is still on the MVP stage and does not provide all the features the Glean SDK does. Feature parity will be worked on after initial validation. Do not hesitate to file bugs using the link above if you want to use Glean.js and is missing some key Glean feature.
+
+## Sections
+
+### [Using Glean](./user/index.html)
+
+If you want to start using Glean to report data then this is the section you should read. It explains the first steps of integrating Glean into your project, choosing the right metric type for you, debugging products that use Glean and even Glean's internal error reporting mechanism.
```

Comment:
I think I would turns this around slightly:

```suggestion
In this chapter we describe how to use the Glean SDK in your own libraries and applications.
It explains the first steps of integrating Glean into your project, choosing the right metric type for you, debugging products that use Glean and Glean's built-in error reporting mechanism.

If you want to start using Glean to report data then this is the section you should read.
```

Reply:
Works for me!

One last nitpick: You use `section` for the other parts so I think it'd be better to also say "In this section ..." here. Sorry, that was my mistake to suggest "chapter"